src/scripts/modules/list.js

```diff
@@ -52,50 +50,71 @@ function renderDestination(destination) {
   elDestination.classList.add("gallery-img--container");
 
   // add the correct class to elDestination wrapper
-  if (destination.visited === true) {
-    elDestination.classList.add("visited");
-  } else {
-    elDestination.classList.add("bucketlist");
-  }
+  destination.visited === true
+    ? elDestination.classList.add("visited")
+    : elDestination.classList.add("bucketlist");
```

Comment:
Uh, some refactors... I like it! Another one even better:

```js
const category = destination.visited === true ? "visited" : "bucketList";
elDestination.classList.add(category);
```

Reply:
I was surprised to discover this works without declaring a variable, is there a downside to it?

Reply:
Not really. Usually, a ternary is used to save values (my example), not to execute operations (your example).

But both make the job.